BuildTools.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26808.3004
+VisualStudioVersion = 15.0.26730.16
```

Comment:
Looks fine but why did this version go down?

Reply:
RTM vs internal previews